dev-docs/modules/currency.md

```diff
@@ -299,4 +306,14 @@ If the timeout occurs while bids are still on the queue, they will be skipped ra
 
 Of course, use of Prebid currency feature is optional.
 
+**Why isn't my currency supported in Prebid's file?**
+
+The data in Prebid's hosted currency file comes from a reliable free source that doesn't contain all the countries in the world.
```

Comment:
Suggest: The data in Prebid's hosted currency file comes from a free source. This source is reliable, but it doesn't contain currencies for all countries of the world.